mlrun/projects/pipelines.py

```diff
@@ -291,15 +328,18 @@ def _create_enriched_mlrun_workflow(
                 for function_obj in functions:
                     # we condition within each function since the comparison between the function and
                     # the kfp pod may change depending on the attribute type.
-                    try:
-                        _set_priority_class_name_on_kfp_pod(
-                            kfp_step_template, function_obj
-                        )
-                    except Exception as err:
-                        kfp_pod_name = kfp_step_template.get("name")
-                        logger.warning(
-                            f"Unable to enrich kfp pod {kfp_pod_name}", error=str(err)
-                        )
+                    _set_function_attribute_on_kfp_pod(
+                        kfp_step_template,
+                        function_obj,
+                        "PriorityClassName",
+                        "priority_class_name",
+                    )
+                    _enrich_kfp_pod_security_context(
+                        kfp_step_template,
+                        function_obj,
+                    )
```

Comment:
I think this should be run inside try except clause, because lets take for example that from some reason user set `kfp_pod_user_unix_id` to 0 or something else raises an exception I wouldn't want to fail the next priority class name enrichments because of that. I guess set a try except inside each enrichment function

Reply:
Resolving, talked offline, we want to fail the enrichment when we are in override mode and user id is set to 0.